src/containers/Game/BattleArena.js

```diff
@@ -6,29 +6,77 @@ import HealthBar from '../../components/Game/HealthBar'
 //STYLING
 import './BattleArena.css'
 
+//CLASSES
+//cleric
+const clericidle = require ('../../img/classes/cleric.gif')
+//fighter
+const fighteridle = require ('../../img/classes/fighter.gif')
+const fightermove = require ('../../img/classes/fightermove.gif')
+const fighterattack = require ('../../img/classes/fighterattack.gif')
+//mage
+const mageidle = require ('../../img/classes/mage.gif')
+const magemove = require ('../../img/classes/magemove.gif')
+const mageattack = require ('../../img/classes/mageattack.gif')
+
 export default class BattleArena extends PureComponent {
   static propTypes = {
     background: PropTypes.string.isRequired,
     player1: PropTypes.string.isRequired,
     player2: PropTypes.string.isRequired,
   }
   state = {
-    player1status :'move',
-    player2status :'move'
+    playerStatus :'idle',
   }
 
-CalulatePercentage = (totalDamage) => {
+calulatePercentage = (totalDamage) => {
   const fullHealth = 10
   return ((fullHealth-totalDamage)*(fullHealth))
```

Comment:
Backend will send current hp not total damage.
So can probably just be current hp instead of fullhealth-totalDamage